docs/examples/compute/instance/block.tf

```diff
@@ -1,14 +1,16 @@
-resource "oci_core_volume" "TFBlock0" {
+resource "oci_core_volume" "TFBlock" {
+  count = "${var.NumInstances * var.NumVolumesPerInstance}"
   availability_domain = "${lookup(data.oci_identity_availability_domains.ADs.availability_domains[var.AD - 1],"name")}"
   compartment_id = "${var.compartment_ocid}"
-  display_name = "TFBlock0"
+  display_name = "TFBlock${count.index}"
   size_in_gbs = "${var.DBSize}"
 }
 
-resource "oci_core_volume_attachment" "TFBlock0Attach" {
+resource "oci_core_volume_attachment" "TFBlockAttach" {
+    count = "${var.NumInstances * var.NumVolumesPerInstance}"
     attachment_type = "iscsi"
```

Comment:
If you change the attachment type to `paravirtualized` the volumes will be attached and usable when the customer logs in, it's a better experience.

Reply:
The provider doesn't support paravirtualized volume attachments yet. Will add a note to add this once we have support for it.